docs/image.md

```diff
@@ -141,6 +142,56 @@ To create the notary secret used by kpack for image signing, run the following c
 - `<password>`: The password provided to encrypt the private key.
 - `<hash>.key`: The private key file.
 
+### <a id='cosign-config'></a>Cosign Configuration
```

Comment:
Would it be alright to have `Signing/Verifying builder/clusterBuilder images` as a separate PR such that the next version of kpack could have the initial image signing first? Then, we will work on implementing the `Signing/Verifying builder/clusterBuilder images` portion of the RFC as the upcoming feature cutoff is near for the next release candidate for kpack.